src/tenants/planet/LeaderBoard/components/Stories.tsx

```diff
@@ -1,37 +1,48 @@
 import React, { ReactElement } from 'react';
 import styles from './Stories.module.scss';
 import i18next from '../../../../../i18n';
+import { lang_path } from '../../../../utils/constants/wpLanguages';
 
 const { useTranslation } = i18next;
 
 interface Props {}
 
 export default function Stories({}: Props): ReactElement {
-  const { t, ready } = useTranslation(['planet']);
-  return ready ? (
-    <div className={styles.wrapper}>
-      <div className={styles.container}>
-        <div className={styles.storyCard}>
-          <img src={'/tenants/planet/images/leaderboard/restoreTrees.svg'} />
-          <div className={styles.storyContent}>
-            <h2>{t('planet:restoreTrees')}</h2>
-            <div>
-              <img src={'/tenants/planet/images/leaderboard/playIcon.svg'} />
-              <p>{t('planet:watchVideo')}</p>
-            </div>
+  const { i18n, t, ready } = useTranslation(['planet']);
+  return (
+    <>
+      {ready ? (
+        <div className={styles.wrapper}>
+          <div className={styles.container}>
+            <a className={styles.storyCard} href={`https://a.plant-for-the-planet.org/${lang_path[i18n.language]}/principles`}>
```

Comment:
This leads to a link like https://a.plant-for-the-planet.org/undefined/principles for Portuguese :-( It works as WP is handling this case, but it does not look very nice.

Reply:
I solved it for principles and standards